source/falaise/snemo/datamodels/topology_data.h

```diff
@@ -0,0 +1,130 @@
+/// \file falaise/snemo/datamodels/topology_data.h
+/* Author (s) : Steven Calvez  <calvez@lal.in2p3.fr>
+ *              Xavier Garrido <garrido@lal.in2p3.fr>
+ * Creation date: 2015-03-31
+ * Last modified: 2015-03-31
+ *
+ * Description:  SuperNEMO Topology data model
+ *
+ * History:
+ *
+ */
+
+#ifndef FALAISE_SNEMO_DATAMODELS_TOPOLOGY_DATA_H
+#define FALAISE_SNEMO_DATAMODELS_TOPOLOGY_DATA_H 1
+
+// Third party:
+// - Bayeux/datatools:
+#include <bayeux/datatools/handle.h>
+#include <bayeux/datatools/i_clear.h>
+#include <bayeux/datatools/i_serializable.h>
+#include <bayeux/datatools/i_tree_dump.h>
+#include <bayeux/datatools/properties.h>
+
+// This project:
+#include <falaise/snemo/datamodels/base_topology_pattern.h>
+
+namespace snemo {
+
+namespace datamodel {
+
+/// \brief SuperNEMO topology data model
+class topology_data : public datatools::i_serializable,
```

Comment:
I don't understand the purpose of this class:

* Why would a `topology_pattern` be shared via a handle?
* What is the purpose of the `auxiliaries` data member?

Reply:
- `auxiliaries` are used for the same reason as explained before. You can think it is useless but the experience has shown it is useful and present in all the top data type with have in SuperNEMO datamodel.

- regarding the purpose of the `topology_data` I don't know what to say: this is the main container of topology pattern so what do you propose ?